inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/util/TestFixtures.java

```diff
@@ -65,7 +65,6 @@ public KnowledgeBase buildKnowledgeBase(Project project, String name, Reificatio
         kb.setLabelIri(RDFS.LABEL);
         kb.setPropertyTypeIri(RDF.PROPERTY);
         kb.setDescriptionIri(RDFS.COMMENT);
-        kb.setDefaultLanguage("en");
```

Comment:
I believe that I had added the language there. Why did you remove it?

Reply:
@reckart I have created another Junit test with language enabled.